lib/upipe-modules/upipe_udp.c

```diff
@@ -572,7 +572,7 @@ int upipe_udp_open_socket(struct upipe *upipe, const char *_uri, int ttl,
         else if (bind_addr.ss.ss_family != AF_UNSPEC) {
     normal_bind:
             if (bind(fd, &bind_addr.so, sockaddr_len) < 0) {
-                upipe_err(upipe, "couldn't bind");
+                upipe_err(upipe, "couldn't bind: %m");
```

Comment:
Keep using %m for now

For Windows the error is in WSAGetLastError() and not errno anyway